### DISCUSSION TITLE
I created a livewire in a subdirectory, how do I override asset_base_url

### DISCUSSION THREAD

**Discussion by nicholasas1** [livewire/livewire#4750] [1. Help]
I created a livewire in a subdirectory, like this http://nicholas.com/explorer

Then I have followed the guide contained here https://laravel-livewire.com/docs/2.x/installation#configuring-the-asset-base-url, I changed the asset_url to '/asset' but nothing happens.

Then I tried to change it to 'asset_url' => env('APP_URL','http://nicholas.com/explorer/asset'),
but still getting 404 like this
https://ibb.co/q97CJXK

how to replace correct asset_url please help me

**Comment by joshhanley** [comment]
@nicholasas1 when you say you put Livewire in a subdirectory, do you mean your whole app is in a subdirectory or you created a subdirectory in the `app/Http/Livewire` directory?

**Reply by nicholasas1** [reply]
no, I created a subfolder in NGINX. The root is nicholas.com is wordpress
and for laravel i put it on nicholas.com/explorer

**Comment by joshhanley** [comment]
@nicholasas1 ah ok, did you also update your `APP_URL` in your `.env` file?

I think Livewire also has an app url config option, maybe you need to update that too.

Hope this helps!

**Reply by nicholasas1** [reply]
This is my .env 

`APP_NAME=Laravel
APP_ENV=local
APP_KEY=base64:aZ59Hf9EfJfdqY8LQyCsqf9gGi9N6u6yn6uUboJHgmo=
APP_DEBUG=true
APP_URL=http://nicholas.com/explorer

ASSET_URL = http://nicholas.com/explorer


API_BASE_URL = https://api.strongbee.tech


LOG_CHANNEL=stack
LOG_DEPRECATIONS_CHANNEL=null
LOG_LEVEL=debug

DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=laravel
DB_USERNAME=root
DB_PASSWORD=

BROADCAST_DRIVER=log
CACHE_DRIVER=file
FILESYSTEM_DRIVER=local
QUEUE_CONNECTION=sync
SESSION_DRIVER=file
SESSION_LIFETIME=120

MEMCACHED_HOST=127.0.0.1

REDIS_HOST=127.0.0.1
REDIS_PASSWORD=null
REDIS_PORT=6379

MAIL_MAILER=smtp
MAIL_HOST=mailhog
MAIL_PORT=1025
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null
MAIL_FROM_ADDRESS=null
MAIL_FROM_NAME="${APP_NAME}"

AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1
AWS_BUCKET=
AWS_USE_PATH_STYLE_ENDPOINT=false

PUSHER_APP_ID=
PUSHER_APP_KEY=
PUSHER_APP_SECRET=
PUSHER_APP_CLUSTER=mt1

MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
`

Can you help me?

**Reply by joshhanley** [reply]
@nicholasas1 thanks, and what does your Livewire config file look like?
